docs/notebooks/plot_predictions.ipynb

```diff
@@ -1,986 +1,1458 @@
 {
```

Comment:
<p>What happened here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
If I remember correctly, I interrupted the cell to run another cell in the same notebook. Nothing to worry about here :)